languages/common-lisp/exercises/concept/sameness/sameness.lisp

```diff
@@ -0,0 +1,13 @@
+(defpackage sameness
+  (:use :cl)
+  (:export :robot))
+
+(in-package :sameness)
+
+(defun robot (room-id)
+  "Return a key to use for the doors in the room designated by ROOM-ID."
```

Comment:
Ah yes, this is an example of a documentation string. I agree adding it to the basics exercise would be good.

Reply:
@TheLostLambda See the above discussion. It might also be nice to update your cheesy (pun intended) exercise too :)

Reply:
That would work for me, we could add them there. Otherwise I was planning on adding them in the `function-definition` exercise, but that means using regular comments until then, which we might not want to do. I'm happy with either!